test/fixtures/dialects/tsql/execute.sql

```diff
@@ -22,3 +22,10 @@ EXECUTE sp_addextendedproperty
 @level0name = N'my_schema_name',
 @level1type = N'my_object_type',
 @level1name = N'my_object_name'
+
+
+-- Executing a stored procedure and capturing the RETURN value in a variable
+EXEC @pRes = dbo.ProcTestDefaults;
+EXEC @pRes = dbo.ProcTestDefaults @p1 = DEFAULT;
+EXECUTE @pRes = dbo.ProcTestDefaults;
+EXECUTE @pRes = dbo.ProcTestDefaults @p1 = DEFAULT;
```

Comment:
Make sure files end in newlines:

```suggestion
EXECUTE @pRes = dbo.ProcTestDefaults @p1 = DEFAULT;

```

 Btw we have a [editorconfig](https://editorconfig.org/) file if your IDE supports that (some like VS Code require installing an extension), and also support [pre-commit](https://github.com/sqlfluff/sqlfluff/blob/main/CONTRIBUTING.md#pre-commit-config), both of which will auto correct this.